.github/workflows/build.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     name: PHP ${{ matrix.php }}-${{ matrix.os }}
 
     env:
+      extensions: gd, openssl
```

Comment:
Do we really need these for sending emails?

Reply:
They are needed for tests. For sending emails, there is no.